src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -1711,10 +1713,13 @@ protected Void job(final SQLiteConnection sqliteConnection)
    */
 
   public void registerFunction(
-      @Nonnull final String functionName, @Nonnull final String functionDefinition)
+      @Nonnull final String functionName,
```

Comment:
These parameter names should change, of course, if the encoding field names change.